.vuepress/_redirects

```diff
@@ -12,6 +12,7 @@
 /link/alexa               /docs/ecosystem/alexa/
 /link/google-assistant    /docs/ecosystem/google-assistant/
 /link/homekit             /addons/integrations/homekit/
+/link/openhabcloud        /addons/integrations/openhabcloud/
```

Comment:
Can you also change `openAB` to `openHAB` on [line 6](https://github.com/openhab/website/pull/266/files#diff-994cfe7bb82fe2fc88eb55ceca4a6b6318e706baaa032a792ffb3379a37d5397R6)? :-)